lib/playground.dart

```diff
@@ -537,6 +541,10 @@ class Playground implements GistContainer, GistController {
   /// Perform static analysis of the source code. Return whether the code
   /// analyzed cleanly (had no errors or warnings).
   Future<bool> _performAnalysis() {
+
+    // We only have issues for dart source code.
+    if (context.focusedEditor != "dart") return null;
```

Comment:
Is this necessary? Hopefully we're only calling `_performAnalysis()` when the dart text is mutated (or when the gist is first loaded).